### DISCUSSION TITLE
How can I see conda env in oh my posh theme in powershell?

### DISCUSSION THREAD

**Discussion by kazimovzaman2** [JanDeDobbeleer/oh-my-posh#3578] [Themes]
I have installed oh-my-posh, and I use the blue-owl theme. From the documentation I add these lines of code to the JSON file;

```
{
    "type": "python",
    "style": "powerline",
    "powerline_symbol": "\uE0B0",
    "foreground": "#100e23",
    "background": "#906cff",
    "template": " \uE235 {{ .Full }}"
},
```

![8EPJy](https://user-images.githubusercontent.com/92586852/224625770-ca6620b6-10b2-4be3-bafd-6b427bbf8445.png)

But I can't see my conda environment. I search for it but can't find anything. Can you help me to add my conda env to PowerShell?

I try to add the following line:

{```

    "type": "command",
    "style": "blue",
    "powerline": "left",
    "foreground": "#F0DB4F",
    "background": "#4F5D75",
    "properties": {
        "command": "conda activate base",
        "leading_icon": " "
    }
},
```
But nothing changed. I can't see conda env.

**Comment by JanDeDobbeleer** [comment]
From the [docs](https://ohmyposh.dev/docs/segments/python#properties):

```json
{
    "type": "python",
    "style": "powerline",
    "powerline_symbol": "\uE0B0",
    "foreground": "#100e23",
    "background": "#906cff",
    "template": " \uE235 {{ .Full }} {{ if .Venv }}{{ .Venv }}{{ end }}",
    "properties": {
        "fetch_virtual_env": true
    }
},
```

**Reply by kazimovzaman2** [reply]
oh thank you, it is work. 🙂
